camlp4/Camlp4/Struct/Lexer.mll

```diff
@@ -208,6 +208,7 @@ module Make (Token : Sig.Camlp4Token)
   let identchar =
     ['A'-'Z' 'a'-'z' '_' '\192'-'\214' '\216'-'\246' '\248'-'\255' '\'' '0'-'9']
   let ident = (lowercase|uppercase) identchar*
+  let quote_tag = (lowercase|uppercase) (identchar|'.')*
```

Comment:
Yeah, it's for simplicity. I didn't want to make the grammar more complex. The user (which is me) will have to validate the tag himself. I think that, when dealing with Camlp4, It's more important not to break anything rather than cater for convenience.